<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWind</class>
 <widget class="QDialog" name="MainWind">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="0" alignment="Qt::AlignLeft|Qt::AlignBottom">
    <widget class="QPushButton" name="openCamera">
     <property name="minimumSize">
      <size>
       <width>92</width>
       <height>0</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>WaitCursor</cursorShape>
     </property>
     <property name="text">
      <string>打开摄像头</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="train">
     <property name="text">
      <string>开始训练</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3" alignment="Qt::AlignRight|Qt::AlignBottom">
    <widget class="QPushButton" name="closeCamera">
     <property name="text">
      <string>关闭摄像头</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="nameInput">
     <property name="text">
      <string>填写姓名</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="takePhoto">
     <property name="text">
      <string>人脸录入</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" rowspan="4" colspan="4">
    <widget class="QLabel" name="showCamerLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="exit">
     <property name="text">
      <string>退出</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>openCamera</sender>
   <signal>clicked()</signal>
   <receiver>MainWind</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>takePhoto</sender>
   <signal>clicked()</signal>
   <receiver>MainWind</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>492</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeCamera</sender>
   <signal>clicked()</signal>
   <receiver>MainWind</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>574</x>
     <y>492</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>train</sender>
   <signal>clicked()</signal>
   <receiver>MainWind</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>548</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nameInput</sender>
   <signal>returnPressed()</signal>
   <receiver>MainWind</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exit</sender>
   <signal>clicked()</signal>
   <receiver>MainWind</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>492</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
